packages/tpetra/core/src/Tpetra_MultiVector_def.hpp

```diff
@@ -4397,7 +4407,8 @@ namespace Tpetra {
     if (useHostVersion) {
       this->modify_host ();
       if (contig || isConstantStride ()) {
```

Comment:
Not quite sure why we're checking both `contig` and `isConstantStride()` here, but that's a separate issue, unrelated to this PR.